ray_on_golem/reputation/updater.py

```diff
@@ -18,10 +20,11 @@ class ReputationUpdater:
 
     def __init__(self, network: str = "polygon"):
         self._network = network
-
-    @property
-    def reputation_uri(self):
-        return f"{REPUTATION_SYSTEM_URI}{REPUTATION_SYSTEM_PROVIDER_SCORES}?network={self._network}"
+        self.reputation_uri = (
+            URL(REPUTATION_SYSTEM_URI)
```

Comment:
Instead of parsing `yarl.URL` here, let's just keep it as a value of constant instead - as we do for many other constants - rich object instead of raw naive value.